settings.gradle

```diff
@@ -1,2 +1,18 @@
-include 'quasar-core','quasar-actors','quasar-reactive-streams','quasar-disruptor'
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    plugins {
+        id 'com.dorongold.task-tree' version "${taskTreeVersion}"
```

Comment:
You can probably remove the quotes and token expansion because the expanded token is the entire string.
```
id 'com.dorongold.task-tree' version taskTreeVersion
```
But that's a Groovy thing.

Reply:
Unfortunately not, I tried that, I get the following error.

`
FAILURE: Build failed with an exception.

* Where:
Settings file '/Users/paulhatcher/R3/github/quasar/settings.gradle' line: 7

* What went wrong:
A problem occurred evaluating settings 'quasar'.
> Could not find method $() for arguments [settings_yth2ytqpo5wrvt5huijaat6t$_run_closure1$_closure3$_closure4@e96af52] on object of type org.gradle.plugin.management.internal.DefaultPluginManagementSpec$PluginDependenciesSpecImpl.
`

Reply:
> Unfortunately not, I tried that, I get the following error. Version has to be a quoted string.
> 
> ```
> FAILURE: Build failed with an exception.
> 
> * Where:
> Settings file '/Users/paulhatcher/R3/github/quasar/settings.gradle' line: 7
> 
> * What went wrong:
> A problem occurred evaluating settings 'quasar'.
> > Could not find method $() for arguments [settings_yth2ytqpo5wrvt5huijaat6t$_run_closure1$_closure3$_closure4@e96af52] on object of type org.gradle.plugin.management.internal.DefaultPluginManagementSpec$PluginDependenciesSpecImpl.
> ```

It works fine in the DJVM's settings.gradle file:
```

Reply:
ok, my bad, done now.